src/onboarding/steps.ts

```diff
@@ -23,12 +23,10 @@ import { StatsigExperiments } from 'src/statsig/types'
 export const END_OF_ONBOARDING_SCREENS = [Screens.WalletHome, Screens.ChooseYourAdventure]
 
 interface NavigatorFunctions {
-  navigate: typeof NavigationService.navigate | ((screen: Screens) => void)
```

Comment:
this dummy type (for calculating total steps) wasn't allowing setting screen, updated the fn below to match navigate's type